GoSSB/GoSSB.xcodeproj/project.pbxproj

```diff
@@ -170,7 +170,6 @@
 			outputFileListPaths = (
 			);
 			outputPaths = (
-				"${PROJECT_DIR}/Sources/.goroot",
```

Comment:
@mplorentz @martindsq 

I had a problem where the script wouldn't be triggered if `.goroot` already existed, can you tell me if there is a better fix for this?

Reply:
These input and output paths are checked on incremental builds, and if none of them have changed then the script isn't run. I'm not sure if removing .goroot from the outputPaths really changed anything, since .goroot will generally only change if the go version in go_install.sh changes. go_install.sh only needs to run if go_install.sh has changed, right?

Reply:
As far as I can tell the script isn't correctly executed unless I remove this line from `.pbxproj`.